layouts/partials/section-index.html

```diff
@@ -9,7 +9,9 @@
         <ul>
             {{ range $pages }}
                 {{ if eq .Parent $parent }}
-                    <li><a href="{{ .RelPermalink }}">{{- .Title -}}</a></li>
+                    {{ $manualLink := cond (isset .Params "manuallink") .Params.manualLink ( cond (isset .Params "manuallinkrelref") (relref . .Params.manualLinkRelref) .RelPermalink) }}
+                    {{ $manualLinkTitle := cond (isset .Params "manuallinktitle") .Params.manualLinkTitle .Title }}
+                    <li><a href="{{ $manualLink }}" titel="{{ $manualLinkTitle }}"{{ with .Params.manualLinkTarget }} target="{{ . }}"{{ if eq . "_blank" }} rel="noopener"{{ end }}{{ end }}>{{- .Title -}}</a></li>
```

Comment:
Should this (and throughout) be `title=` rather than `titel=`

Reply:
Yes, of course. Thanks!
This is a typically mistake when you are a native German: the German word for title is Titel. ;-)

Reply:
Heh, I assumed that was the case!

Reply:
Changed titel to title in https://github.com/google/docsy/pull/511/commits/7273745ae1b1aebe151190d4763fe6899f9f7a69